src/ActiveField.php

```diff
@@ -113,6 +113,21 @@ class ActiveField extends \yii\widgets\ActiveField
      * {@inheritdoc}
      */
     public $inputOptions = ['class' => ['widget' => 'form-control']];
+    /**
+     * @var array the default options for the input tags (checkboxes and radios). The parameter passed to individual
+     * input methods (e.g. [[textInput()]]) will be merged with this property when rendering the input tag.
+     *
+     * If you set a custom `id` for the input element, you may need to adjust the [[$selectors]] accordingly.
+     *
+     * @see \yii\helpers\Html::renderTagAttributes() for details on how attributes are being rendered.
+     * @since 2.0.7
+     */
+    public $checkOptions = [
```

Comment:
Templates are separated for checkboxes and radios. It is likely a good idea to have options separated as well.